### PR TITLE
Schemaform field documentation

### PR DESCRIPTION
**Pull request by jbalboni**
Trying to add some more documentation for the vets.gov specific fields we have.

### CODE REVIEW

File: docs/schemaform/field-library.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+This is for our common EFT information field, where we collect account type (checking/savings), bank account number, and routing number.
+
+### Currency
+- File: src/js/common/schemaform/definitions/currency.js
+- uiSchema: Yes
+- schema: No
+- Function or object: Function
+
+### Current or past dates
```
Comment (cehsu) [inline docs/schemaform/field-library.md]: Is it beyond the scope to mention that we also validate by specific ranges?

**Review by cehsu — APPROVED** [review]
LGTM. This is really great. Would it make sense to list any libraries we use with this such as Downshift? Are strategies we've developed, such as prefilling certain fields/defaulting to zero, within the scope of this documentation?

**Review by rianfowler — APPROVED** [review]
This is useful.

File: docs/schemaform/field-library.md (hunk 1 above)
Comment (jbalboni) [reply docs/schemaform/field-library.md]: I'm not sure what you mean

Comment (cehsu) [reply docs/schemaform/field-library.md]: For example, veterans need to be born at least 15 years prior to the current date for certain forms.

File: docs/schemaform/field-library.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+Widget       | Default schema type or format
+------------ | -------------
+TextWidget | type: string
+SelectWidget | type: string with an enum property
+RadioWidget |
+CheckboxWidget | type: boolean
+yesNo |
+
+There are more widgets that just these, but we normally use definitions they're included in, rather than just the field. These are the ones you're likely to have to set at some point in a config file.
```
Comment (annekainicUSDS) [inline docs/schemaform/field-library.md]: Small typo: "There are more widgets **than** just these"

**Review by annekainicUSDS — APPROVED** [review]
LGTM!

File: docs/schemaform/field-library.md (hunk 1 above)
Comment (jbalboni) [reply docs/schemaform/field-library.md]: I think those are mostly form specific (and we have a pattern in here for personId), so I think we can leave them out.

**Pull request merged**